creation/create_cvmfsexec_distros.sh

```diff
@@ -177,8 +181,8 @@ fi
 
 # after confirming that the remaining number of arguments to be either 1 or 2
 re_sources="^((osg|egi|default),)*(osg|egi|default),?$"
-# TODO: update the regex for machine types upon checking with `makedist -h` periodically
-re_mtypes="^((rhel(7|8)|suse15)(-x86_64),?)+$"
+# TODO: update the regex for rhel9 and other machine types upon checking with `makedist -h` periodically
+re_mtypes="^((rhel(7|8)|suse15)(-(x86_64|aarch64|ppc64le),?))+$"
```

Comment:
Also here rhel9  should be in the allowed types. We are not blocking it, we are providing a warhing that it does not work in el7

Reply:
Updated as suggested.

**Note to self:** We still need to include el9 support for cases where cvmfsexec distributions is required to be generated on an el9 machine, even though doing so on an el7 machine fails (this detail has been added in the CHANGELOG suggesting to exercise caution when generating distributions for el9 on an el7 machine).